### ISSUE TITLE
Remove sensitive information

### ISSUE THREAD

**Issue by lylythechosenone** [is-a-dev/register#11329]
### Your subdomain
lylythechosenone.is-a.dev
### The issue
It appears I've accidentally released two pieces of personal information (my email address and my IP address) into this repository. My IP address has already been changed by my ISP, so that's less of an issue (although the domain it's attached to, vpn.lylythechosenone.is-a.dev was a mistake anyway, and I'd like to remove it). However, I would like for my personal email to not be displayed in the commit history at all. I'm working on making an email forwarded through this domain instead, but I'm not sure how I would go about removing my personal email from the commit history.

TLDR: I accidentally added my personal email to this repo, and I'd like for it to be erased and replaced with `lysander@lylythechosenone.is-a.dev`.

**Comment by wdhdev** [comment]
As you have committed your email to the repository and multiple forks, etc have been made since then, we cannot help you unfortunately. You can open a PR to change it to something else if you would like.
